### PR TITLE
Display db

### PR DESCRIPTION
**Pull request by Skylucas**
I have made it so the about me section from my database appears on screen to the server, i have made a admin page that uses functions, but i have also made the tests that pass both a success and fail, I have 2 separate pages, one called admin that makes the functions and the other one called querys_admin that will upload the data onto the server.

### CODE REVIEW

**Review by jaholt — COMMENTED** [review]
I actually can't see anything wrong.

File: admin_functions.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<?php
+
```
Comment (charliecog) [inline admin_functions.php:2]: line break not needed

File: admin_functions.php [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+/*
+ * This function will select the about me sections and the text from the the about content table. Will then fetch the query and execute that query
+ *
+ * @param parameter for the input would be $section
+ *
+ * @return this will return the string that I have defined as a section.
+ */
+
+function fetchInput($db) {
```
Comment (charliecog) [inline admin_functions.php]: type hint param to PDO

File: admin_functions.php [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    $query = $db->prepare("SELECT `about_me_sections`, `text` FROM `about_content`;");
+    $query->execute();
+    $section = $query->fetch();
+    return $section;
+}
+
+/*
+ * This function fetches all the keys that I need from my database, and uses a if statement where if the key exists from the database return the key otherwise return an error.
+ *
+ * @param parameter for the input would be $sdbResults and $key
```
Comment (charliecog) [inline admin_functions.php]: need to detail your params individually

File: admin_functions.php [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+ *
+ * @return this will return the dbResults for key and $key that picks any key from the database.
```
Comment (charliecog) [inline admin_functions.php]: have a look at what doc blocks should look like eg: data types etc..

File: dbConn.php [hunk 5]
```diff
@@ -0,0 +1,7 @@
+<?php
+
+function dbConn(){
```
Comment (charliecog) [inline dbConn.php:8]: doc block?

File: portfolio.php [hunk 6]
```diff
@@ -1,3 +1,8 @@
+<?php
+
```
Comment (charliecog) [inline portfolio.php]: take this php tags out if not using

File: querys_admin.php [hunk 7]
```diff
@@ -0,0 +1,13 @@
+<?php
+
+require_once 'dbConn.php';
+require_once 'admin_functions.php';
+
+$db = dbConn();
+
+$about_content = fetchInput($db);
+$about_result = fetchContent('text', $about_content);
+//var_dump($about_result);
```
Comment (charliecog) [inline querys_admin.php]: remove var dump line

File: querys_admin.php [hunk 8]
```diff
@@ -0,0 +1,13 @@
+<?php
+
+require_once 'dbConn.php';
+require_once 'admin_functions.php';
+
+$db = dbConn();
+
+$about_content = fetchInput($db);
+$about_result = fetchContent('text', $about_content);
```
Comment (charliecog) [inline querys_admin.php]: this is a duplicate of line 12 remove if not needed

File: test/test_about_me.php [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+
+require_once('../admin_functions.php');
+
+use PHPUnit\Framework\TestCase;
+
+class admin extends TestCase {
+//
+//    public function testfetchUrlsSuccess() {
+//        $expected = "Pilot shop/pilotshop.html";
+//        $testarray = ["text"=>"My name is Dan and I am a Trainee Full Stack Developer"];
```
Comment (charliecog) [inline test/test_about_me.php]: take it out if not using it (any commented out code)

File: test/test_about_me.php [hunk 10]
```diff
@@ -0,0 +1,39 @@
...
+        $expected = "Error incorrect input";
+        $testarray = ["text"=>"My name is Dan and I am a Trainee Full Stack Developer"];
+        $case = fetchContent('poop', $testarray);
+        $this->assertEquals($case, $expected);
+
+    }
+
+//    public function testfetchUrlsFailures() {
+//        $expected = "Error incorrect input";
+//        $testarray = ["text"=>"Pilot shop/pilotshop.html"];
```
Comment (charliecog) [inline test/test_about_me.php]: remove

**Pull request merged**